packages/pangea-sdk/pangea/request.py

```diff
@@ -230,6 +230,10 @@ def post(
         if isinstance(data, BaseModel):
             data = data.model_dump(exclude_none=True)
 
+        if isinstance(data, dict):
+            # Remove `None` values.
+            data = {k: v for k, v in data.items() if v is not None}
```

Comment:
How do we fall into this case?

Reply:
It's for my `"analyzers": analyzers` addition in the Prompt Guard client, with `analyzers` being `None` by default.

Reply:
Oh, got it, we don't have a `BaseModel` request implemented there. It would be handled by `data.model_dump(exclude_none=True)` in that case.